test/e2e/common/expansion.go

```diff
@@ -30,6 +30,8 @@ import (
 // These tests exercise the Kubernetes expansion syntax $(VAR).
 // For more information, see:
 // https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/expansion.md
+//
+// TODO(#98326): Is this sig-node or sig-storage?
```

Comment:
@dchen1107 @derekwaynecarr - can you please advise on it?

[it's also possible that split by SIG maybe at the level of individual tests here]

Reply:
SIG Node please.

Reply:
thanks - done